<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="test_button">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>13</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>253</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Сохранить список позиций по счетам</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_position_data">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>43</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>253</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Отобразить позиции</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_accounts">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>73</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>253</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Обновить список счетов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_instruments">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>103</y>
     <width>253</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>253</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Обновить список инструментов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
